apps/backend/package.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "antalmanac-backend",
+    "version": "0.0.1",
+    "description": "Backend for AntAlmanac",
+    "main": "server.js",
+    "scripts": {
+        "build": "esbuild --platform=node --target=node16 --bundle  --outfile=dist/lambda.js lambda.js",
+        "start": "node server.local.js",
+        "format": "prettier --write \"./**/*.{ts,tsx,js}\""
```

```suggestion
        "format": "prettier --write src"
```
I usually just do something like this and call it a day lol